plugins/org.eclipse.osee.orcs.rest/src/org/eclipse/osee/orcs/rest/internal/RelationEndpointImpl.java

```diff
@@ -91,6 +92,27 @@ public List<RelationTypeToken> convertRelations(ArtifactId artToken, RelationTyp
       return null;
    }
    
+   @Override
+   public List<RelationTypeToken> convertAllRelations(RelationTypeToken oldRelationType,
+      RelationTypeToken newRelationType) {
+      RelationTypeSide sideB = new RelationTypeSide(oldRelationType, RelationSide.SIDE_B);
+      TransactionBuilder tx = orcsApi.getTransactionFactory().createTransaction(branch,
+         String.format("Converting relations for Old Relation = %s New Relation = %s", oldRelationType.getName(),
+            newRelationType.getName()));
+
+      for (ArtifactReadable art : orcsApi.getQueryFactory().fromBranch(branch).andRelationExists(
+         oldRelationType).getResults().getList()) {
+         if (art.getRelatedCount(sideB) > 0) {
+            for (ArtifactReadable artB : art.getRelated(sideB).getList()) {
+               tx.relate(art, newRelationType, artB);
+            }
+         }
+      }
+      tx.commit();
+
+      return null;
```

Comment:
Q: Can we make this return the results of tx.commit()?

Reply:
of course, I should have caught that.